lib/newrelic-fluentd-output/version.rb

```diff
@@ -1,3 +1,3 @@
 module NewrelicFluentdOutput
-  VERSION = "1.2.0"
+  VERSION = "1.3.0"
```

Comment:
Hi 👋 wondering if these change should be a Patch. What do You think 🤔

Reply:
Yup, I agree, a patch/bugfix bump should be enough, thanks!